extra/1-currency-conversion.js

```diff
@@ -15,7 +17,9 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(amount) {
+  return Math.floor((amount * 0.99 * 5.7) * 100) / 100;
```

Comment:
Here, for better readability, you should separate this computation into separate variables, for example:
 - What is the transaction fee?
 - What's the exchange rate?
 - what's the amount after the fee?

It may look like too much for a simple computation, but the idea with these exercises is to start practicing your code readability skills that will be very useful in the future :)